notebooks/Making a Custom Statespace Model.ipynb

```diff
@@ -0,0 +1,1464 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        x0 = pm.Deterministic(
</pre><p>Do these need to be <code>Deterministic</code> for some of the conditional methods? Otherwise it would be more intuitive to write them as <code>ConstantData</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-experimental/pull/174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
They're not really data though. It's closest to a Dirac at the specified initial value. In the purest sense these should be estimated, but in practice usually they aren't, so I wanted to show both.